packages/contracts/test/registry/dao-registry.ts

```diff
@@ -7,6 +7,7 @@ import {deployNewDAO} from '../test-utils/dao';
 import {deployENSSubdomainRegistrar} from '../test-utils/ens';
 import {SignerWithAddress} from '@nomiclabs/hardhat-ethers/signers';
 import {deployWithProxy} from '../test-utils/proxy';
+import { Wallet } from 'ethers';
```

Comment:
Still problems with prettier?

Reply:
yeah no clue why the default option isn't taken -.-